examples/imitation_learning_openbot/run_trained_policy.py

```diff
@@ -220,8 +185,8 @@
             if args.debug:
                 show_obs(
                     obs,
-                    config.SIMULATION_CONTROLLER.OPENBOT_AGENT.OBSERVATION_COMPONENTS,
-                    config.SIMULATION_CONTROLLER.OPENBOT_AGENT.CAMERA.RENDER_PASSES)
+                    config.SIMULATION_CONTROLLER.VEHICLE_AGENT.OBSERVATION_COMPONENTS,
+                    config.SIMULATION_CONTROLLER.VEHICLE_AGENT.CAMERA.RENDER_PASSES)
             
             # termination conditions
             if env_step_info["task_step_info"]["hit_obstacle"]: 
```

Comment:
- mimic the `if args.benchmark ... else ...` structure in `generate_dataset.py`

Reply:
I don't think we need a `else` condition here as we do not have to worry about data collection here.